cocos2d/CCDirector.h

```diff
@@ -66,13 +66,13 @@
 /** Stops the animation. Nothing will be drawn. The main loop won't be triggered anymore.
  If you want to pause your animation call [pause] instead.
  */
--(void) stopAnimation;
+-(void) stopRunLoop;
 
 /** The main loop is triggered again.
- Call this function only if [stopAnimation] was called earlier
- @warning Don't call this function to start the main loop. To run the main loop call runWithScene
+ Call this function only if [stopRunLoop] was called earlier
+ @warning Don't call this function to start the main loop. To run the main loop call presentScene
  */
--(void) startAnimation;
+-(void) startRunLoop;
```

Comment:
I would recommend renaming this, as having a method `startRunLoop` along with the attached warning that states "don't call this to start the loop" is pretty counter intuitive